### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by hanckmann** [dddomodossola/remi#204]
Hi,
I am planning to use remi in my projects, and I like what I see so far. The one thing which is missing is good documentation.
There is a list of widgets and after that... the only effective way to learn more is to look into the source code and into the examples (also source code of course)
It would be a real improvement when good documentation would be available.

**Comment by dddomodossola** [comment]
You are right, however don't underevaluate the support chat at https://gitter.im/dddomodossola/remi
